### DISCUSSION TITLE
Why we need to build curl from source code to use oqs-provider?

### DISCUSSION THREAD

**Discussion by kaiduanx** [open-quantum-safe/oqs-demos#293] [General]
On Ubuntu 20.04, I built and installed openssl/liboqs/oqsprovder, and set the LD_LIBRARY_PATH to openssl with oqsprovider support, however system installed curl still does not use the OpenSSL library with oqsprovider support. Any reason why? Is there any way to use system installed curl **without rebuild it from source** to use oqs-provider? This question also applies to other application such as OpenVPN/nginx etc.

**Comment by baentsch** [comment]
That might depend on the distro build of curl I'd think. What are you using?
Edit/add: Ubuntu 20 (I should read everything, sorry). I'm using 22 but could give 20 a try too when I'm back at home next week.
